src/app/dynamic/enterprise/metering/schedule-config/add-dialog/template.html

```diff
@@ -0,0 +1,103 @@
+  </km-dialog-title>
+
+  <mat-dialog-content>
+    <form [formGroup]="form"
+          fxFlex
+          fxLayout="column"
+          fxLayoutGap="10px">
+
+      <mat-form-field fxFlex>
+        <mat-label>Schedule Name</mat-label>
+        <input required
+               matInput
+               [formControlName]="controls.Name"
+               type="text"
+               autocomplete="off"
+               cdkFocusInitial>
+        <mat-error *ngIf="form.get(controls.Name).hasError('required')">
+          Schedule Name is <strong>required</strong>.
+        </mat-error>
+      </mat-form-field>
+
+      <mat-radio-group [formControlName]="controls.Group"
+                       fxLayout="column"
+                       fxLayoutGap="20px">
+        <mat-radio-button [value]="ScheduleOption.Daily">
+          <div class="km-radio-button-title">Daily</div>
+          <div class="km-radio-button-description">Create report every day at 01:00</div>
+        </mat-radio-button>
+
+        <mat-radio-button [value]="ScheduleOption.Weekly">
+          <div class="km-radio-button-title">Weekly</div>
+          <div class="km-radio-button-description">Create report every week at 01:00 on Saturday</div>
+        </mat-radio-button>
+
+        <mat-radio-button [value]="ScheduleOption.Monthly">
+          <div class="km-radio-button-title">Monthly</div>
+          <div class="km-radio-button-description">Create report every month at 01:00 on 1st day of the month</div>
+        </mat-radio-button>
+
+        <mat-radio-button [value]="ScheduleOption.Custom">
+          <div fxFlex
+               fxLayout="column">
+            <div class="km-radio-button-title">Custom Configuration</div>
+            <div class="km-radio-button-description">Set a custom report schedule.</div>
+          </div>
+        </mat-radio-button>
+      </mat-radio-group>
+
+      <div *ngIf="form.get(controls.Group).value === ScheduleOption.Custom"
+           fxLayout="column"
+           fxLayoutGap="20px">
+        <mat-form-field>
+          <mat-label>Cron Expression</mat-label>
+          <input [formControlName]="controls.Schedule"
+                 type="text"
+                 matInput
+                 required>
+          <mat-hint>
+            Cron expression that describes how often a report should be created. Must match the criteria specified
+            <a href="https://pkg.go.dev/github.com/robfig/cron?utm_source=godoc"
+               target="_blank">here</a>.
+          </mat-hint>
+        </mat-form-field>
+
+        <km-number-stepper [formControlName]="controls.Interval"
+                           mode="errors"
+                           label="Interval"
+                           hint="Scope of the report (in days)."
+                           min="1"
+                           required>
+        </km-number-stepper>
+      </div>
+    </form>
+  </mat-dialog-content>
+
+  <mat-dialog-actions>
+    <button mat-flat-button
```

Comment:
Please use the km-button component that we have added recently. Check https://github.com/kubermatic/dashboard/pull/4363 and https://github.com/kubermatic/dashboard/pull/4396 for reference.